"Roles" page no response

### Page on which it happened
index.php?page=users

### Steps to reproduce
1. Click "Roles" function

### Expected behaviour
Add/Delete/Edit Roles

### Actual behaviour
The "Roles" web page is freezing, no response on any links.

### Server configuration
**Operating system**: Linux  5.4.0-100-generic #113-Ubuntu SMP Thu Feb 3 18:43:29 UTC 2022 x86_64

**Web server:** Apache/2.4.41 (Ubuntu)

**Database:** 5.5.5-10.3.34-MariaDB-0ubuntu0.20.04.1

**PHP version:** 7.4.28

**Teampass version:** 3.0.0.14

**Teampass configuration file:**
```
'max_latest_items' => '10',
'enable_favourites' => '1',
'show_last_items' => '1',
'enable_pf_feature' => '0',
'log_connections' => '1',
'log_accessed' => '1',
'time_format' => 'H:i:s',
'date_format' => 'd/m/Y',
'duplicate_folder' => '0',
'item_duplicate_in_same_folder' => '0',
'duplicate_item' => '0',
'number_of_used_pw' => '3',
'manager_edit' => '1',
'cpassman_dir' => '/var/www/html/teampass',
'cpassman_url' => 'http://<anonym_url>
'favicon' => 'http://<anonym_url>/favicon.ico',
'path_to_upload_folder' => '/var/www/html/teampass/upload',
'path_to_files_folder' => '/var/www/html/teampass/files',
'url_to_files_folder' => 'http://192.168.8.83/files',
'activate_expiration' => '0',
'pw_life_duration' => '0',
'maintenance_mode' => '0',
'enable_sts' => '0',
'encryptClientServer' => '1',
'cpassman_version' => '3.0.0.13',
'ldap_mode' => '0',
'ldap_type' => '0',
'ldap_suffix' => '0',
'ldap_domain_dn' => '0',
'ldap_domain_controler' => '0',
'ldap_user_attribute' => '0',
'ldap_ssl' => '0',
'ldap_tls' => '0',
'ldap_elusers' => '0',
'ldap_search_base' => '0',
'ldap_port' => '389',
'richtext' => '0',
'allow_print' => '1',
'roles_allowed_to_print' => '0',
'show_description' => '1',
'anyone_can_modify' => '0',
'anyone_can_modify_bydefault' => '0',
'nb_bad_authentication' => '0',
'utf8_enabled' => '1',
'restricted_to' => '0',
'restricted_to_roles' => '0',
'enable_send_email_on_user_login' => '1',
'enable_user_can_create_folders' => '1',
'insert_manual_entry_item_history' => '0',
'enable_kb' => '0',
'enable_email_notification_on_item_shown' => '0',
'enable_email_notification_on_user_pw_change' => '0',
'custom_logo' => '',
'custom_login_text' => '',
'default_language' => 'english',
'send_stats' => '0',
'send_statistics_items' => 'stat_country;stat_users;stat_items;stat_items_shared;stat_folders;stat_folders_shared;stat_admins;stat_managers;stat_ro;stat_mysqlversion;stat_phpversion;stat_teampassversion;stat_languages;stat_kb;stat_suggestion;stat_customfields;stat_api;stat_2fa;stat_agses;stat_duo;stat_ldap;stat_syslog;stat_stricthttps;stat_fav;stat_pf;',
'send_stats_time' => '1643900300',
'get_tp_info' => '1',
'send_mail_on_user_login' => '0',
'nb_items_by_query' => 'auto',
'enable_delete_after_consultation' => '0',
'enable_personal_saltkey_cookie' => '0',
'personal_saltkey_cookie_duration' => '31',
'email_smtp_server' => '<removed>'
'email_smtp_auth' => '1',
'email_auth_username' => '<removed>'
'email_auth_pwd' => '<removed>'
'email_port' => '587',
'email_security' => 'tls',
'email_server_url' => '',
'email_from' => '<removed>'
'email_from' => '<removed>'
'pwd_maximum_length' => '40',
'google_authentication' => '0',
'delay_item_edition' => '0',
'allow_import' => '1',
'proxy_ip' => '',
'proxy_port' => '',
'upload_maxfilesize' => '10mb',
'upload_docext' => 'doc,docx,dotx,xls,xlsx,xltx,rtf,csv,txt,pdf,ppt,pptx,pot,dotx,xltx',
'upload_imagesext' => 'jpg,jpeg,gif,png',
'upload_pkgext' => '7z,rar,tar,zip',
'upload_otherext' => 'sql,xml',
'upload_imageresize_options' => '1',
'upload_imageresize_width' => '800',
'upload_imageresize_height' => '600',
'upload_imageresize_quality' => '90',
'use_md5_password_as_salt' => '0',
'ga_website_name' => 'TeamPass for ChangeMe',
'api' => '0',
'subfolder_rights_as_parent' => '1',
'show_only_accessible_folders' => '0',
'enable_suggestion' => '0',
'otv_expiration_period' => '7',
'default_session_expiration_time' => '60',
'duo' => '0',
'enable_server_password_change' => '0',
'ldap_object_class' => '0',
'bck_script_path' => '/var/www/html/teampass/backups',
'bck_script_filename' => 'bck_teampass',
'syslog_enable' => '0',
'syslog_host' => 'localhost',
'syslog_port' => '514',
'manager_move_item' => '1',
'create_item_without_password' => '1',
'otv_is_enabled' => '0',
'agses_authentication_enabled' => '0',
'item_extra_fields' => '0',
'saltkey_ante_2127' => 'none',
'migration_to_2127' => 'done',
'files_with_defuse' => 'done',
'timezone' => 'Asia/Hong_Kong',
'enable_attachment_encryption' => '1',
'personal_saltkey_security_level' => '50',
'ldap_new_user_is_administrated_by' => '0',
'disable_show_forgot_pwd_link' => '0',
'offline_key_level' => '0',
'enable_http_request_login' => '0',
'ldap_and_local_authentication' => '0',
'secure_display_image' => '1',
'upload_zero_byte_file' => '0',
'upload_all_extensions_file' => '0',
'bck_script_passkey' => '<removed>'
'admin_2fa_required' => '1',
'password_overview_delay' => '4',
'copy_to_clipboard_small_icons' => '1',
'duo_akey' => '',
'duo_ikey' => '',
'duo_skey' => '',
'duo_host' => '',
'teampass_version' => '',
'roles_allowed_to_print_select' => '[1]',
'clipboard_life_duration' => '30',
'mfa_for_roles' => '',
'tree_counters' => '0',
'settings_offline_mode' => '0',
'settings_tree_counters' => '0',
'enable_massive_move_delete' => '1',
'email_debug_level' => '1',
'can_create_root_folder' => '1',

```

**Updated from an older Teampass or fresh install:**

### Client configuration

**Browser:** Chrome - 99.0.4844.51

**Operating system:** Windows - 64bits

### Logs

#### Web server error log
```
[Sun Mar 13 16:48:01.306858 2022] [php7:notice] [pid 48069] [client 192.168.8.107:54625] PHP Notice:  Undefined offset: 25 in /var/www/html/teampass/pages/roles.php on line 103, referer: http://192.168.8.83/index.php?page=folders
[Sun Mar 13 16:48:01.307411 2022] [php7:notice] [pid 48069] [client 192.168.8.107:54625] PHP Notice:  Trying to access array offset on value of type null in /var/www/html/teampass/pages/roles.php on line 103, referer: http://192.168.8.83/index.php?page=folders
[Sun Mar 13 16:48:01.307770 2022] [php7:error] [pid 48069] [client 192.168.8.107:54625] PHP Fatal error:  Uncaught TypeError: addslashes() expects parameter 1 to be string, null given in /var/www/html/teampass/pages/roles.php:103\nStack trace:\n#0 /var/www/html/teampass/pages/roles.php(103): addslashes()\n#1 /var/www/html/teampass/index.php(992): include('/var/www/html/t...')\n#2 {main}\n  thrown in /var/www/html/teampass/pages/roles.php on line 103, referer: http://192.168.8.83/index.php?page=folders

```

#### Teampass 10 last system errors
```

```

#### Log from the web-browser developer console (CTRL + SHIFT + i)
```
csrfprotector.js:152 TypeError: Cannot read properties of null (reading 'value')
    at Object._init (csrfprotector.js:150:86)
    at csrfprotector_init (csrfprotector.js:174:11)
    at csrfprotector.js:336:5
_init @ csrfprotector.js:152
csrfprotector.js:153 [ERROR] [CSRF Protector] unable to parse blacklisted url fields.
_init @ csrfprotector.js:153
index.php:1          Failed to load resource: the server responded with a status of 500 (Internal Server Error)
```

--- a/install/install.queries.php
+++ b/install/install.queries.php
@@ -916,7 +916,7 @@ $SETTINGS = array (';
                         if ($tmp === 0) {
                             $mysqli_result = mysqli_query(
                                 $dbTmp,
-                                "INSERT INTO `" . $var['tbl_prefix'] . "roles_title` (`id`, `title`, `allow_pw_change`, `complexity`, `creator_id`) VALUES (NULL, 'Default', '0', '50', '0')"
+                                "INSERT INTO `" . $var['tbl_prefix'] . "roles_title` (`id`, `title`, `allow_pw_change`, `complexity`, `creator_id`) VALUES (NULL, 'Default', '0', '48', '0')"
                             );
                         }
                     } elseif ($task === 'roles_values') {

